modules/drivers/mongo/src/metabase/driver/mongo/util.clj

```diff
@@ -70,7 +70,7 @@
          skip (.skip skip)
          batch-size (.batchSize (int batch-size))
          sort-criteria (.sort (mongo.conversion/to-document (ordered-map/ordered-map sort-criteria))))
-       (mapv #(mongo.conversion/from-document % opts))))
+       (map #(mongo.conversion/from-document % opts))))
```

Comment:
there are more places here that use `mapv`. My gut says we should convert them all. Should we? @lbrdnk

Reply:
I'll open tech debt issue tracking this.